src/main/java/teammates/storage/entity/BaseEntity.java

```diff
@@ -1,7 +1,47 @@
 package teammates.storage.entity;
 
+import java.time.Instant;
+import java.util.Date;
+
+import com.googlecode.objectify.impl.Path;
+import com.googlecode.objectify.impl.translate.CreateContext;
+import com.googlecode.objectify.impl.translate.LoadContext;
+import com.googlecode.objectify.impl.translate.SaveContext;
+import com.googlecode.objectify.impl.translate.SkipException;
+import com.googlecode.objectify.impl.translate.TypeKey;
+import com.googlecode.objectify.impl.translate.ValueTranslator;
+import com.googlecode.objectify.impl.translate.ValueTranslatorFactory;
+
 /**
  * Base class for all entities persisted to the Datastore.
  */
 public abstract class BaseEntity { // NOPMD
+
+    /**
+     * Translates between `java.time.Instant` in entity class and `java.util.Date` in Google Cloud Datastore.
```

Comment:
Am I right to understand that this can be removed once Objectify is at 6?

Reply:
The code is still needed :( as Objectify V6 doesn't ship with the above translator in its distributed package. However, the dependency of `java.util.Date` can be removed because a [new set of GAE API](https://cloud.google.com/datastore/docs/concepts/entities#date_and_time) is adopted in Objectify V6 (The reason why `java.util.Date` is still here is because the old set of [GAE API use it](https://cloud.google.com/appengine/docs/standard/java/datastore/entities#Java_Properties_and_value_types)).